src/app/cli/src/init/client.ml

```diff
@@ -866,36 +866,47 @@ let dump_keypair =
       |> Public_key.Compressed.to_base58_check )
       (kp.private_key |> Private_key.to_base58_check))
 
+let handle_dump_ledger_response ~json = function
+  | Error e ->
+      Daemon_rpcs.Client.print_rpc_error e
+  | Ok (Error e) ->
+      printf !"Ledger not found: %s\n" (Error.to_string_hum e)
+  | Ok (Ok accounts) ->
+      if json then
+        List.iter accounts ~f:(fun acct ->
+            printf "%s\n"
+              (Yojson.Safe.to_string (Account.Stable.Latest.to_yojson acct)) )
+      else printf !"%{sexp:Account.t list}\n" accounts
```

Comment:
Could you dump these in `Runtime_config.Accounts.t` format? It is a bit more user friendly and allows simply copying the output of the command to config json that the daemon uses for genesis ledger and epoch ledgers. This function `Genesis_ledger_helper.Accounts.Single.of_account` does the conversion

Reply:
Or a flag that determines what format to output in (Account.t by default)? I don't know if it is used in any tooling.

Reply:
Done